src/kontrol/solc_to_k.py

```diff
@@ -776,7 +777,19 @@ def __init__(self, contract_name: str, contract_json: dict, foundry: bool = Fals
                 for rng in ref:
                     self.immutable_ranges.append((rng['start'], rng['length']))
 
+        self.link_ranges = []
+        if 'linkReferences' in deployed_bytecode:
+            for ref in deployed_bytecode['linkReferences'].values():
+                for rng_grp in ref.values():
+                    for rng in rng_grp:
+                        self.link_ranges.append((rng['start'], rng['length']))
+
         self.deployed_bytecode = deployed_bytecode['object'].replace('0x', '')
+        try:
+            bytearray.fromhex(self.deployed_bytecode)
+        except ValueError:
+            print(f'non hex value for deployed_bytecode found in {self.name_with_path}')
+            print(self.deployed_bytecode)
```

Comment:
Just a small comment here. Perhaps it would be better to use `_LOGGER.error` instead of `print` for consistency?

Reply:
Yes, I am getting some prints while testing that probably shouldn't be there.

Reply:
Fixed.